chrome/content/zotero/preferences/preferences_export.js

```diff
@@ -133,35 +139,55 @@ Zotero_Preferences.Export = {
 	},
 	
 	
-	updateQuickCopyHTMLCheckbox: function () {
+	updateQuickCopyUI: function () {
 		var format = document.getElementById('zotero-quickCopy-menu').value;
+		
 		var mode, contentType;
 		
-		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		[mode, format] = format.split('=');
 		[mode, contentType] = mode.split('/');
 		
+		var checkbox = document.getElementById('zotero-quickCopy-copyAsHTML');
 		checkbox.checked = contentType == 'html';
 		checkbox.disabled = mode != 'bibliography';
+		
+		var menulist = document.getElementById('locale-menu');
+		if (mode != 'bibliography') {
+			// maintain menu selection but show blank label
+			menulist.insertItemAt(0, '', menulist.value);
+			menulist.selectedIndex = 0;
+			menulist.disabled = true;
```

Comment:
Well, the menulist should be disabled, but the label should remain black/dark. I would take that to suggest that the selected locale still applies, but is not mutable. When you select an export translator, the locale setting is irrelevant as is the "Copy as HTML" setting. The "Copy as HTML" checkbox _and_ label are both grayed out, but for the locale menulist, the label remains black. I think that looks odd.